src/main/resources/templates/records.html

```diff
@@ -3,81 +3,86 @@
+                    </div>
+                </details>
             </div>
-          </details>
+            <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
         </div>
-        <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
-      </div>
-      <div class="table-wrapper">
-        <table th:if="${!content.records.isEmpty()}">
-          <thead>
-            <tr>
-              <th scope="col">
-                <a th:href="${linkResolver.resolve(new uk.gov.register.core.RegisterName('field'), registerId)}" th:text="${registerId}"></a>
-              </th>
-              <th:block th:each="fieldName : ${register.nonPrimaryFields}">
-                <th scope="col">
-                  <a th:href="${linkResolver.resolve(new uk.gov.register.core.RegisterName('field'), fieldName)}" th:text="${fieldName}"></a>
-                </th>
-              </th:block>
-            </tr>
-          </thead>
-          <tbody>
-            <tr th:each="record : ${content.records}">
-              <td>
-                <a th:href="${'/record/' + record.primaryKey}" th:text="${record.primaryKey}"></a>
-              </td>
-              <th:block th:each="fieldName : ${register.nonPrimaryFields}">
-                <td th:if="${record.getField(fieldName).present}" class="field-list field-list-in-table">
-                  <ul th:if="${record.getField(fieldName).get().isList()}">
-                    <th:block th:each="fieldValue: ${record.getField(fieldName).get()}">
-                      <li th:text="${fieldValue.value}"></li>
+        <div class="table-wrapper">
+            <table th:if="${!content.records.isEmpty()}">
+                <thead>
+                <tr>
+                    <th scope="col">
+                        <a th:href="${linkResolver.resolve(new uk.gov.register.core.RegisterName('field'), registerId)}"
+                           th:text="${registerId}"></a>
+                    </th>
+                    <th:block th:each="fieldName : ${register.nonPrimaryFields}">
+                        <th scope="col">
+                            <a th:href="${linkResolver.resolve(new uk.gov.register.core.RegisterName('field'), fieldName)}"
+                               th:text="${fieldName}"></a>
+                        </th>
                     </th:block>
-                  </ul>
-                  <th:block th:unless="${record.getField(fieldName).get().isList()}" th:text="${record.getField(fieldName).get().value}">
-                  </th:block>
-                </td>
-                <td th:unless="${record.getField(fieldName).present}"></td>
-              </th:block>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+                </tr>
+                </thead>
+                <tbody>
+                <th:block th:each="record : ${content.records}">
+                    <tr th:each="item : ${record.itemViews}">
```

Comment:
True - I just wanted to do something that works at this stage. We might need some help from front end to make it look nice.